File: src/main/java/com/sudosoo/takeiteasy/service/HeartService.java
package com.sudosoo.takeiteasy.service;

import com.sudosoo.takeiteasy.dto.heart.CommentHeartRequestDto;
import com.sudosoo.takeiteasy.dto.heart.PostHeartRequestDto;
import com.sudosoo.takeiteasy.entity.Heart;

public interface HeartService {

    Heart createPostHeart(PostHeartRequestDto heartRequestDTO);
    Heart createCommentHeart(CommentHeartRequestDto heartRequestDTO);
    void postDisHeart(PostHeartRequestDto heartRequestDTO);
    void commentDisHeart(CommentHeartRequestDto heartRequestDTO);


}

File: takeItEasy-application/src/main/kotlin/com/sudosoo/takeItEasy/service/HeartService.kt
package com.sudosoo.takeItEasy.service

import com.sudosoo.takeiteasy.dto.heart.CommentHeartRequestDto

interface HeartService {
    fun createPostHeart(heartRequestDTO: PostHeartRequestDto): Heart
    fun createCommentHeart(heartRequestDTO: CommentHeartRequestDto): Heart
    fun postDisHeart(heartRequestDTO: PostHeartRequestDto)
    fun commentDisHeart(heartRequestDTO: CommentHeartRequestDto)
}
